src/index.scss

```diff
@@ -0,0 +1,5 @@
+@import 'carbon-components/scss/globals/scss/styles.scss';
+
+$feature-flags: (
```

Comment:
This mapping should come before the styles import. 
https://www.carbondesignsystem.com/tutorial/react-step-1#add-ui-shell-sass